src/interface/eager.jl

```diff
@@ -51,6 +51,26 @@ function Base.reduce(op::Function, bc::Broadcasted{FinchStyle{N}}; dims=:, init
     end
 end
 
+
+function tensordot(A::Tensor, B::Tensor, idxs; kw...)
+    bc_A = broadcasted(identity, A)
```

Comment:
Can we instead do `compute(tensordot(lazy(A), lazy(B), idxs, kw...))`?

Reply:
Done.